tests/phpunit/PaymentMethods/WC_Stripe_UPE_Payment_Gateway_Test.php

```
@@ -505,6 +505,64 @@ public function test_payment_fields_outputs_fields() {
+		$registered_script = wp_scripts()->registered['wc-stripe-upe-classic'] ?? null;
+
+		// Clean up registered scripts/styles and the filter so subsequent tests are not affected.
+		remove_filter( 'woocommerce_is_checkout', '__return_true' );
+		wp_deregister_script( 'wc-stripe-upe-classic' );
+		wp_deregister_style( 'wc-stripe-upe-classic' );
+
+		$this->assertTrue( $script_is_registered, 'wc-stripe-upe-classic script is not registered' );
+		$this->assertNotNull( $registered_script, 'wc-stripe-upe-classic script is not a valid object' );
+		$this->assertSame( $expected_version, $registered_script->ver, 'wc-stripe-upe-classic script version is not the same as the expected version' );
+		$this->assertSame( $expected_dependencies, $registered_script->deps, 'wc-stripe-upe-classic script dependencies are not the same as the expected dependencies' );
+	}
```

PHPMD (2.15.0)

[error] The method test_payment_scripts_registers_script_with_correct_version is not named in camelCase. (undefined)

(CamelCaseMethodName)